app/metrics/Monitors.scala

```diff
@@ -61,4 +64,7 @@ object Monitors {
       case s if s <= 500 => Counter("arrivals-per-eori-251-500")
       case _             => Counter("arrivals-per-eori-501-or-more")
     }
+
+  def countMessagesFromNcts(messageType: MessageType, channel: ChannelType, outcome: SubmissionProcessingResult): Counter =
```

Comment:
This can be used more generically, possibly on the out bound side. For example we can count number of IE015 vs IE014 (assuming that's a useful metric).

So this could just be (in words) _count the number of messages by the message type_

Reply:
Good point!  I'll take a look at whether this can be applied to the outbound stuff too.  You're right that it might be useful to graph them all together.